src/start.js

```diff
@@ -24,6 +22,10 @@ export const startServer = ({ port = process.env.PORT } = {}) => {
     }
   }
 
+  app.get('/ping/success', (req, res) => res.json({ message: 'pong' }))
```

Comment:
casco anche io spesso nella tentazione di tornare un espressione invece che il blocco di codice nelle arrow functions, ma attenzione che così rischi di tornare cose che a volte non vorresti, personalmente quando ho il dubbio, apro il blocco di codice senza return, così impongo `undefined` come valore di ritorno.

In questo caso non so res.json(...) se torna qualcosa (e vabè, in questo caso specifico potrebbe non fregarci nulla comunque, essendo un handler di express...)

Reply:
ok capito